sdks/paths.mk

```diff
@@ -42,10 +42,8 @@ ifndef DISABLE_IOS
 
 CheckXcodeDir=$(or $(and $(wildcard $(1))),$(warning Could not find Xcode in "$(1)"))
 
-ifneq ($(call CheckXcodeDir,/Applications/Xcode.app/Contents/Developer),)
-XCODE_DIR=/Applications/$(XCODE_VERSION).app/Contents/Developer
-else
-$(error Could not find Xcode)
+ifeq ($(wildcard $(XCODE_DIR)),)
```

Comment:
Could we just keep the call to `CheckXcodeDir` here or does it breal something?

Reply:
What does that call actually do here ?

Reply:
So I think we should error out if the xcode path doesn't exist, thats why the call was removed.

Reply:
It keeps it consistent with the Android NDK and SDK check. We do fail in the previous implementation, right after printing the warning.